pkg/print/common/errors.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package common
+
+import "fmt"
+
+// ResultError is returned from printers when the apply/destroy operations completed, but one or
+// more resources either failed apply/prune/delete, or failed to reconcile.
+type ResultError struct {
+	ApplyFailed  int
+	PruneFailed  int
+	DeleteFailed int
+
+	ReconcileTimeout int
+	ReconcileFailed  int
+}
+
+func (a *ResultError) Error() string {
+	failedActuateSum := a.ApplyFailed + a.PruneFailed + a.DeleteFailed
```

Comment:
Are clients going to need these sums? Should they be a method on the type?

Reply:
The idea was to not add additional methods for now. We can always add more methods if needed.